src/Symfony/Component/Yaml/Inline.php

```diff
@@ -32,19 +32,42 @@ class Inline
      *
      * @param string $value                  A YAML string
      * @param bool   $exceptionOnInvalidType true if an exception must be thrown on invalid types (a PHP resource or object), false otherwise
-     * @param bool   $objectSupport          true if object support is enabled, false otherwise
-     * @param bool   $objectForMap           true if maps should return a stdClass instead of array()
+     * @param int    $flags                  A bit field of PARSE_* constants to customize the YAML parser behavior
      * @param array  $references             Mapping of variable names to values
      *
      * @return array A PHP array representing the YAML string
      *
      * @throws ParseException
      */
-    public static function parse($value, $exceptionOnInvalidType = false, $objectSupport = false, $objectForMap = false, $references = array())
+    public static function parse($value, $exceptionOnInvalidType = false, $flags = 0, $references = array())
```

Comment:
Would it make sense to also include the `$exceptionOnInvalidType` argument into the flags?